apps/angular-app/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
There is a `.editorconfig` in the root file, I think this one is not necessary

Reply:
I have deleted the ".editorconfig" file from angular project. The only difference to highlight was the rule for single quotes in .ts file